Error 'Unable to locate enclosing class' when executing clirr API checks

**Describe the bug**

The clirr plugin reports errors when executed on module `model-apis/xml-model`.

```
11:31:40,279 [INFO] --- clirr:2.8:check-no-fork (restrictive) @ operaton-xml-model ---
11:31:40,280 [INFO] Comparing to version: 1.0.0
11:31:40,332 [ERROR] Unable to locate enclosing class org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest for nested class org.operaton.bpm.model.xml.test.AbstractModelElementInstanceTest$AttributeAssumption
1
```

The error does not break the build.

**To Reproduce**

Steps to reproduce the behavior:

```
mvn clean install -Pcheck-api-compatibility -pl model-api/xml-model -am
```

**Expected behavior**

No error in the build log.

**Additional context**
The class `AbstractModelElementInstanceTest`has been recently moved with commit `79e85a278f828f058876e7b977af5d18734f9d88`

--- a/model-api/xml-model/pom.xml
+++ b/model-api/xml-model/pom.xml
@@ -72,6 +72,7 @@
               <excludes>
                 <exclude>org/operaton/bpm/model/xml/impl/**</exclude>
                 <exclude>org/operaton/bpm/model/xml/test/AbstractModelElementInstanceTest</exclude>
+                <exclude>org/operaton/bpm/model/xml/test/AbstractModelElementInstanceTest$*</exclude>
               </excludes>
             </configuration>
             <executions>

